### DISCUSSION TITLE
HwV380E31_WF6_PTZ_WIFI

### DISCUSSION THREAD

**Discussion by Clashonpot** [Arkady23/V380-Web#63] [General]
![IMG_20230814_024310](https://github.com/Arkady23/V380-Web/assets/142177899/3ed2770a-ece1-4fcd-8ffe-e35ea84d7d40)

![IMG_20230814_024503](https://github.com/Arkady23/V380-Web/assets/142177899/aaa67071-34ae-4704-9af9-322bf71b24c8)

i use the latest file (Initial installer • version 17-07-2023)

quick_check_dump.txt: https://paste.onl/?6bc79228209fa38a#FKzcqhLDrqp5r4sUJVcwVYgZP13wJFAvq2EhTvvBKsnX

**Comment by Clashonpot** [comment]
[upgrade_V380_AK_3918E-V330_DMA-20M_V9.9.9.9_2022-02-17.zip](https://github.com/Arkady23/V380-Web/files/12332117/upgrade_V380_AK_3918E-V330_DMA-20M_V9.9.9.9_2022-02-17.zip)

**Comment by Clashonpot** [comment]
[ceshi.ini.zip](https://github.com/Arkady23/V380-Web/files/12332131/ceshi.ini.zip)

**Comment by Barbar16** [comment]
I installed this update on my own camera, and now my camera is not working :)

**Comment by Arkady23** [comment]
A new way to install a webcam has appeared. I'll describe it on the main page soon.
